### PR TITLE
[GITHUB-20] Update to Latest Standards

### PR DESCRIPTION
**Pull request by dennisconrad**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -2,31 +2,29 @@
 
 dependencies:
   - role: sansible.java
-    version: v1.0
-    java:
-      version: "{{ logstash.java.version }}"
+    sansible_java_vendor: oracle
```
Comment (lobsterdore) [inline meta/main.yml]: Why Oracle and not the default?

Comment (dennisconrad) [reply meta/main.yml]: Good catch!  Leftover from testing.

File: requirements.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 
 - src: sansible.java
-  version: v1.0
-  
+  version: v2.0.1
```
Comment (lobsterdore) [inline requirements.yml]: Should be v2.0 (and users and groups below).

File: tasks/install.yml [hunk 3]
```diff
@@ -1,12 +1,16 @@
 ---
 
-- name: "Install 2.x Family"
+- name: Install apt-transport-https
```
Comment (lobsterdore) [inline tasks/install.yml:3]: Is this step needed? If yes then it probably needs become: yes.

**Comment by lobsterdore** [comment]
Small comment, looks great otherwise.

File: tasks/install.yml (hunk 3 above)
Comment (dennisconrad) [reply tasks/install.yml:3]: It should be installed by default, but wasn't on the docker images.  Required for `https://` repos.  Added the `become: yes`.

**Comment by lobsterdore** [comment]
LGTM 👍

**Review by rsgoodall — APPROVED** [review]

**Pull request merged**